python-client/build.gradle.kts

```diff
@@ -87,5 +87,15 @@ tasks {
         module = "pdm"
         command = "build"
     }
+
+    create("sphinx-build") {
+        dependsOn("install")
+        doLast {
+            exec {
+                commandLine("./.venv/bin/sphinx-build", "docs", "docs/_build/html")
```

Comment:
I think we should follow the existing call hierarchy: gradle -> pdm -> task
like here:

```
    create<PythonTask>("sphinx-build") {
        dependsOn("install")
        module = "pdm"
        command = "run sphinx-build"
    }
```

in this case it'll be easy to run the same commands directly with pdm (which will happen once we do the repo split and get rid of gradle for python-client)